app/screens/receive-bitcoin-screen/receive-usd.tsx

```diff
@@ -343,6 +354,8 @@ const ReceiveUsd = () => {
   }, [invoice?.paymentRequest])
 
   const copyToClipboard = () => {
+    if (!invoice?.paymentRequest) return
```

Comment:
Maybe show an error toast here?

Reply:
Yeah I was debating what to do here, lets discuss it in mobile sync. It should technically not happen, but we dont have the type safety.

Reply:
maybe add a `// TODO` so someone eventually looks at it?

Reply:
I think the correct behavior is to make a function null if the function is not able to perform its core tasks because of null | undefined dependencies. This way we dont have functions floating around that don't do what their name says they do.

Reply:
@nb1 @daviroo thoughts on this? I just pushed the new changes